src/components/Lobby/Room/Room.jsx

```diff
@@ -10,12 +10,12 @@ export class Room extends React.Component {
         const waitingVisible = !this.props.room.started && (this.props.room.minPlayers - this.props.room.players.length > 0);
         return (
             <div className={styles.room} key={this.props.room.roomId} data-cy="room">
-                <h2 className={styles.roomHeader} data-cy="room-title">{"Room: " + this.props.room.roomId}</h2>
+                <h2 className={styles.roomHeader} data-cy="room-title">{"Room: " + this.props.room.title}</h2>
                 <ul className={styles.roomPlayers} id="players" data-cy='room-players'>
                     {this.props.room.players.map((player, key) => (
                         <li className={styles.playerInRoom} key={key}>
                             <span className={styles.roomPlayer}data-cy='player-username'>{player.username}</span>
-                            {player.username.includes("Computer") && <button className={styles.removeAIButton} onClick={() => this.props.removeAIPlayer(this.props.room.roomId, player.username)} data-cy='remove-ai'>-</button>}
+                            {player.username.includes("CPU") && <button className={styles.removeAIButton} onClick={() => this.props.removeAIPlayer(this.props.room.roomId, player.username)} data-cy='remove-ai'>-</button>}
```

Comment:
What happens if someone has CPU in their username?

Reply:
That might be an issue, let me fix this